src/frameworks/drupal8/solr.md

```diff
@@ -12,7 +35,9 @@ $ composer require drupal/search_api drupal/search_api_solr
 
 And then commit the changes to `composer.json` and `composer.lock`.
 
-## Configuration
+Now, enable the modules in your Platform website via the Drupal UI or via a command line tool like Drush or Drupal Console.
+
+### Configuration
```

Comment:
This is the section that needs to be replaced with the copy-paste-able code.  We should specifically not be telling people to use the UI.